pages/care.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style/style.css">
+    <link rel="stylesheet" href="/style/care.css">
+    <title>Care</title>
+    <link rel="icon" href="../assets/plant-icon.png" type="image/icon">
+</head>
+<body>
+    <header>
+      <h1>Plant Party Fansite</h1>
+      <nav>
+        <ul>
+          <li>
+            <a class="menu-link" href="../index.html">Home</a>
+          </li>
+          <li>
+            <a class="menu-link" href="#">Care</a>
+          </li>
+          <li>
+            <a class="menu-link" href="facts.html">Facts</a>
+          </li>
+          <li>
+            <a class="menu-link" href="gallery.html">Gallery</a>
+          </li>
+        </ul>
+      </nav>
+  </header>
+  <main>
+    <div class="intro">
+      <div id="summary">
+          <p> This is a quick guide on how to take care of your plants!</p>
+      </div>
+      <div id="table-of-contents">
```

Comment:
Prefer using `section` here since it's more semantic. It seems like `div` is used to wrap a section that's related to table of contents. However, since `div` isn't semantic, we can use `section` here.